README.md

```diff
@@ -423,17 +471,13 @@ install to the name you `import` is not always self-evident:
   dependency.
 
 To solve this, FawltyDeps looks at the packages installed in your current Python
-environment to correctly map dependencies (package names) into the imports that
-they provide.
-
-However, when a declared dependency is _not_ found in your current environment,
-FawltyDeps will fall back to an "identity mapping", that is, we will _assume_
-that when you depend on `some_package`, then that should correspond to an
-`import some_package` statement in your code.
+environment (or the virtualenv given by the `--venv` option) to correctly map
+dependencies (package names) into the imports that they provide.
 
-This fallback assumption is not always correct, but it allows FawltyDeps to
-produce results (albeit sometimes inaccurate) when the current Python
-environment does not contains all of your declared dependencies.
+However, when an installed package is not found for a declared dependency, the
+_identity mapping_ that FawltyDeps falls back to will still do an _OK_ job for
```

Comment:
```suggestion
_identity mapping_ that FawltyDeps falls back to will still do a good job for
```
FD is doing an _OK_ job on a set of random non installed dependencies, but for the majority where the import name is the package name, this is not just an OK job but a good one. So we either have to replace Ok with something else or reword the sentence to make it clear that the Ok adjective applies to all non found dependencies rather than the majority where identity mapping actually works great.